### PR TITLE
Mf/param updater update pos global params

### PR DESCRIPTION
**Pull request by mattfoley8**
Goes with https://github.com/deso-protocol/backend/pull/496.

### CODE REVIEW

File: lib/block_view_stake.go [hunk 1]
```diff
@@ -1955,7 +1955,7 @@ func (bav *UtxoView) IsValidUnlockStakeMetadata(transactorPkBytes []byte, metada
 	}
 
 	// Validate EndEpochNumber + StakeLockupEpochDuration <= CurrentEpochNumber.
-	if metadata.EndEpochNumber+bav.Params.StakeLockupEpochDuration > currentEpochNumber {
+	if metadata.EndEpochNumber+bav.GetStakeLockupEpochDuration(0) > currentEpochNumber {
```
Comment (lazynina) [inline lib/block_view_stake.go:1958]: Should we be fetching the StakeLockupEpochDuration based on a snapshot?

File: lib/block_view_validator.go [hunk 2]
```diff
@@ -1715,7 +1715,7 @@ func (bav *UtxoView) IsValidUnjailValidatorMetadata(transactorPublicKey []byte)
 	}
 
 	// Validate sufficient epochs have elapsed for validator to be unjailed.
-	if validatorEntry.JailedAtEpochNumber+bav.Params.ValidatorJailEpochDuration > currentEpochNumber {
+	if validatorEntry.JailedAtEpochNumber+bav.GetValidatorJailEpochDuration(0) > currentEpochNumber {
```
Comment (lazynina) [inline lib/block_view_validator.go:1718]: same thing here about using a snapshot

File: lib/block_view_stake.go (hunk 1 above)
Comment (mattfoley8) [reply lib/block_view_stake.go:1958]: Yes. We should be fetching those based on a snapshot. For now, I'm just keeping the existing behavior. I'll update exactly which snapshot we should fetch once I know it. That'll just be swapping out the `0` for the correct EpochNumber relative to the CurrentEpochNumber.

File: lib/block_view_validator.go (hunk 2 above)
Comment (mattfoley8) [reply lib/block_view_validator.go:1718]: Yup, you are correct. Same response as above.

File: lib/block_view_stake.go (hunk 1 above)
Comment (mattfoley8) [reply lib/block_view_stake.go:1958]: Yes. We should be fetching those based on a snapshot. For now, I'm just keeping the existing behavior. I'll update exactly which snapshot we should fetch once I know it. That'll just be swapping out the `0` for the correct EpochNumber relative to the CurrentEpochNumber.

**Review by lazynina — APPROVED** [review]
Just check that extra data isn't nil, but otherwise this looks good.

**Pull request merged**